cli/src/cluster_query.rs

```diff
@@ -1389,7 +1434,11 @@ pub fn process_ping(
 
         let build_message = |lamports| {
             let ix = system_instruction::transfer(&config.signers[0].pubkey(), &to, lamports);
-            Message::new(&[ix], Some(&config.signers[0].pubkey()))
+            if let Some(compute_budget_ix) = build_compute_budget_instruction(additional_fee) {
+                Message::new(&[ix, compute_budget_ix], Some(&config.signers[0].pubkey()))
+            } else {
+                Message::new(&[ix], Some(&config.signers[0].pubkey()))
+            }
```

Comment:
think it just needs to be in the first 3 IXs: https://github.com/solana-labs/solana/blob/master/program-runtime/src/compute_budget.rs#L112-L115

Reply:
Ah, "otherwise ignored."  I  must've missed that during compute budget review, or else I'd have complained :sweat_smile: 

@jackcmay should `ComputeBudget` instructions be prefixed by convention even though we aren't enforcing it?